app/views/charityortrust/trust/add/StartDateView.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.charityortrust.trust.add.routes._
+
+@this(
+    main_template: MainTemplate,
+    formHelper: FormWithCSRF
+)
+
+@(form: Form[_], name: String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("trustBeneficiary.startDate.title")}"
+) {
+
+    @formHelper(action = StartDateController.onSubmit(), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.heading("trustBeneficiary.startDate.heading", Some(name))
+
+        <p>@messages("trustBeneficiary.startDate.paragraph")</p>
+
+        @components.input_date(
+            field = form("value"),
+            hint = Some(messages("trustBeneficiary.startDate.hint"))
+        )
+
+        @components.submit_button()
+
+        <p><a href="@controllers.routes.AddABeneficiaryController.onPageLoad().url">@messages("site.return.link")</a></p>
```

Comment:
The add a beneficiary links should not be on the pages yet as you cannot have an "in progress" beneficiary. This is not just scoped to this story. A bit of tech debt.